mmv1/third_party/terraform/tests/resource_bigtable_gc_policy_test.go

```diff
@@ -30,6 +30,7 @@ func TestAccBigtableGCPolicy_basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccBigtableGCPolicyExists(
 						t, "google_bigtable_gc_policy.policy"),
+						resource.TestCheckResourceAttr("google_bigtable_gc_policy.policy", "gc_rules", ""),
```

Comment:
Verify not just gc_rules, compare in string if the gc_rules matches with predefined string.